scripts/verify-install.sh

```diff
@@ -49,22 +64,23 @@ if [[ $# -eq 2 ]]; then
     exit 1
   fi
 elif [[ $# -eq 0 ]]; then
-  # Make sure both binaries can be found and are executable.
-  check_command lnd
-  check_command lncli
+  # By default we're picking up lnd and lncli from the system $PATH.
+  LND_BIN=$(which lnd)
+  LNCLI_BIN=$(which lncli)
 else
   echo "ERROR: invalid number of parameters!"
   echo "Usage: verify-install.sh [lnd-binary lncli-binary]"
   exit 1
 fi
 
+# Make sure both binaries can be found and are executable.
+check_command lnd
```

Comment:
This should be `check_command "$LND_BIN"` now. Otherwise it will check if there's an `lnd` in your `$PATH`.